packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js

```diff
@@ -7,7 +7,7 @@ import {
   object,
   optionsKnob as options,
 } from "@storybook/addon-knobs";
-import { SfHeader, SfLink } from "@storefront-ui/vue";
+import { SfHeader, SfLink, SfMegaMenu, SfList, SfMenuItem, SfBottomNavigation } from "@storefront-ui/vue";
```

Comment:
There is an error occurring in console:
![image](https://user-images.githubusercontent.com/32042425/86597894-a4dd6e80-bf9c-11ea-9e51-379de8d34c0f.png)

so maybe importing observer will be good solution?
import {
  mapMobileObserver,
} from "../../../utilities/mobile-observer";

Reply:
I can't see this error :/

Reply:
Maybe it is connected with my local project :)